analyzers/src/SonarAnalyzer.Common/SymbolicExecution/Roslyn/ProgramState.cs

```diff
@@ -122,26 +124,26 @@ public ProgramState ResetOperations()
         public override int GetHashCode() =>
             // VisitCount is not included, it's not part of Equals
             HashCode.Combine(
-                Exception?.GetHashCode() ?? 0,
                 HashCode.DictionaryContentHash(OperationValue),
                 HashCode.DictionaryContentHash(SymbolValue),
+                HashCode.DictionaryContentHash(CaptureOperation),
                 HashCode.EnumerableContentHash(PreservedSymbols),
-                HashCode.DictionaryContentHash(CaptureOperation));
+                HashCode.EnumerableContentHash(Exceptions));
```

Comment:
Maybe we should order these alphabetically:
```
            HashCode.Combine(
                HashCode.DictionaryContentHash(CaptureOperation),
                HashCode.DictionaryContentHash(OperationValue),
                HashCode.DictionaryContentHash(SymbolValue),
                HashCode.EnumerableContentHash(Exceptions),
                HashCode.EnumerableContentHash(PreservedSymbols));
```

Reply:
I have them in the order as they are defined in the class. Like that, it's easy to match them 1:1 that no one is missing.

And I've fixed the order in `Equals` now 🤦